analysers/analyser_osmosis_relation_duplicate_member.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Osmose Project 2023                                        ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from modules.OsmoseTranslation import T_
+from .Analyser_Osmosis import Analyser_Osmosis
+
+sql10 = """
+SELECT
+  id,
+  ST_AsText(relation_locate(id)),
+  array_agg(duplicate)
+FROM (
+  SELECT
+    id,
+    member_type || member_id || ' (' || COUNT(*) || ')' AS duplicate
```

Comment:
I think it is possible to report the duplicated member object using `any_full`.

Reply:
I was considering this at first (well, with `array_*` instead of `any_*` to avoid many reports for the same relation if there's more than one member duplicated, which is true for about half the cases I tested), but I think it would become confusing as soon as there's more than one duplicate member.

If so the UI would show something like:
```
relation 123
way 123
way 456
way 789
```
which would be confusing because way 123, 456 and 789 are not duplicates. Hence I output it as text, for example: `w123 (2), w456 (3), w789 (2)`.

I would do this if it was possible to repeat the entries that were duplicated, e.g. 
```
relation 123
way 123
way 123
way 456
way 456
way 456
way 789
way 789
```
In that case it would be clear to the user that those are three different repeated members.

Reply:
I'll check in the coming days if I can combine the two approaches.

Reply:
Done :)